Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM ubuntu
-MAINTAINER marcelstoer
+MAINTAINER mikeller
```

Comment:
This was not meant to be included, it is a commit that I made after the pull request, when I pushed the script to the docker registry. Stripped.